index.html

```diff
@@ -73,9 +73,9 @@ <h2>Seconds</h2>
           <!-- Update activityType.innerText to activity category -->
           <p id="activityType">Activity Type</p>
           <!-- input parsed countdown time to countdownTimerText on each interval -->
-          <span id="countdownTimerText">05:00</span>
+          <span id="countdownTimerText"></span>
           <!-- classlist.add on this with the below VV format based on category selected-->
-          <div id="startTimerButton" class="meditate-start-button">START</div>
+          <button id="startTimerButton" class="meditate-start-button">START</button>
```

Comment:
Change class name to something more generic

Reply:
changed to "start-button"